gulpfile.js/tasks/build.js

```diff
@@ -0,0 +1,62 @@
+const { src, dest, parallel } = require('gulp');
+
+const sass = require('gulp-sass');
+const rename = require('gulp-rename');
+const postcss = require('gulp-postcss');
+const cssnano = require('cssnano');
+
+const rollup = require('rollup');
+const path = require('path');
+const multiEntry = require('@rollup/plugin-multi-entry');
+const nodeResolve = require('@rollup/plugin-node-resolve');
+const { terser } = require('rollup-plugin-terser');
+
+sass.compiler = require('sass');
+
+const outDir = 'dist';
+
+const buildSass = () => {
+  return src('./src/index.scss')
+    .pipe(
+      sass({
+        // Import Theo design tokens as SCSS variables
```

Comment:
Is this an accidental duplicate comment, or intentional?

Reply:
That was an accident, I fixed it ✅